blocks/footer/footer.css

```diff
@@ -1,14 +1,258 @@
-footer {
-  padding: 2rem;
-  background-color: var(--light-color);
-  font-size: var(--body-font-size-s);
-}
-
 footer .footer {
-  max-width: 1200px;
+  max-width: 100vw;
   margin: auto;
+  background-color: var(--navy-blue);
+  color: var(--background-color);
+}
+
+.footer ul {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.footer li {
+  font-size: var(--body-font-size-xs);
+  padding-left: 0;
+  padding-bottom: 10px;
+}
+
+.footer-main {
+  display: flex;
+  justify-content: space-between; 
+  flex-flow: row nowrap; 
+  gap: 20px; 
+  padding: 40px 0 64px;
+}
+
+
+.footer-main a {
+  color: var(--background-color);
+  text-decoration: none;
+}
+
+.footer-social-wrapper a, .footer-languages-wrapper a {
+  color: var(--background-color);
+  text-decoration: none;
+}
+
+.footer-sublinks-row a {
+  font-family: var(--avenir-regular);
+  color: var( --med-gray);
+  display: inline-block;
+  padding: 0 20px;
+  text-decoration: none;
+}
+
+.footer-social-wrapper a {
+  color: var(--background-color);
+}
+
+.footer-social a {
+  margin-left: 20px;
+}
+
+.footer a:hover {
+  color: var(--footer-link-hover-color);
+}
+
+.footer-main h3 {
+  font-family: var(--avenir-demi);
+  font-size: var(--heading-font-size-xxs);
+  margin: 0 0 1.25em;
+}
+
+.footer-column {
+  flex-grow: 1; 
+  flex-basis: 25%;
+  max-width: 25%;
+  padding-left: 40px;
+}
+
+.social-languages-container,
+.sublinks-default-container {
+  display: flex;
+  justify-content: space-between;
+  gap: 20px;
+  flex-flow: row wrap;
+  margin-top: 20px;
+}
+
+.footer-social-wrapper,
+.footer-languages-wrapper,
+.footer-sublinks-wrapper {
+  flex: 1;
+  min-width: 200px;
+}
+
+.footer-sublinks-wrapper ul {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  justify-content: center;
+  width: 100%;
+}
+
+.footer-sublinks-row li {
+  font-family: var(--avenir-demi);
+  font-size: var(--body-font-size-xs);
+  color: var(--navy-blue);
+  padding: 0 20px;
 }
 
-footer .footer p {
+.footer-social-row {
+  max-width: 100vw !important;
```

Comment:
Generally you shouldn't have to use !important -- that's usually a sign that something else is taking priority that shouldn't, or that your selectors are not specific enough so they're not getting priority.